packages/hash/search-loader/package.json

```diff
@@ -16,7 +16,7 @@
     "@hashintel/hash-api": "0.0.0",
     "@hashintel/hash-backend-utils": "0.0.0",
     "@hashintel/hash-shared": "0.0.0",
-    "@opensearch-project/opensearch": "1.0.0",
+    "@opensearch-project/opensearch": "1.1.0",
```

Comment:
Looks like upgrading was not necessary, but not worth reverting back to 1.0.0 given that this is already done.

Release notes don't seem to contain anything interesting:
https://github.com/opensearch-project/opensearch-js/releases